### DISCUSSION TITLE
including library plugins without import

### DISCUSSION THREAD

**Discussion by equilet** [katspaugh/wavesurfer.js#3580] [Q&A]
Is it possible to include plugins without a remote `<script>` tag and without using `import`?
I would like to download the library + plugins locally and include via a relative path in my `<script>`.  

For whatever reason, this is not working for me.

**Comment by katspaugh** [comment]
Yes, that should work. Make sure to download the min.js files if you're not using `import`. E.g. https://unpkg.com/wavesurfer.js@7.7.3/dist/wavesurfer.min.js
